pkg/subnet/public.go

```diff
@@ -111,6 +111,42 @@ func (d *PublicDeployer) AddValidator(
 	return false, tx, remainingSubnetAuthKeys, nil
 }
 
+// AddValidatorPrimaryNetwork adds node as Primary Network Validator
+func (d *PublicDeployer) AddValidatorPrimaryNetwork(
+	nodeID ids.NodeID,
+	weight uint64,
+	startTime time.Time,
+	duration time.Duration,
+	recipientAddr ids.ShortID,
+	shares uint32,
+) error {
+	wallet, err := d.loadWallet()
+	if err != nil {
+		return err
+	}
+	validator := &txs.Validator{
+		NodeID: nodeID,
+		Start:  uint64(startTime.Unix()),
+		End:    uint64(startTime.Add(duration).Unix()),
+		Wght:   weight,
+	}
+	if d.usingLedger {
+		ux.Logger.PrintToUser("*** Please sign AddValidatorPrimaryNetwork transaction on the ledger device *** ")
```

Comment:
Please use "Please sign AddValidator transaction on the ledger device" as that is the message in ledger I believe